package/index.js

```
@@ -19,9 +19,9 @@ const globalMutableWebpackConfig = () => {
   return environmentConfig
 }
 
-const generateWebpackConfig = () => {
+const generateWebpackConfig = (extraConfig = {}) => {
   const environmentConfig = globalMutableWebpackConfig()
```

Let's throw an error if there is more than one arg.